### PR TITLE
Selenium: Targeting smoke tests

### PR DESCRIPTION
**Pull request by szymon-kellton**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -218,6 +218,7 @@ jobs:
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
       - name: E2E tests
+        continue-on-error: true
```
Comment (wozniakpl) [inline .github/workflows/ci.yml:221]: do we want the PR to be all green even if e2e tests fail?
and to trigger deploy on e2e failure?

Comment (szymon-kellton) [reply .github/workflows/ci.yml]: Was changed to:
if: always()

Comment (wozniakpl) [reply .github/workflows/ci.yml]: I think we don't need anything there
if: always() will make it run even if previous steps fail I think, e.g. docker build

Comment (wozniakpl) [reply .github/workflows/ci.yml]: oh, nevermind
if: always() in artifacts is ok
I thought you are putting this in the e2e step

**Review by johniak — APPROVED** [review]

**Review by pavlo-mk — DISMISSED** [review]

**Review by pavlo-mk — APPROVED** [review]

**Review by patryk-dabrowski — DISMISSED** [review]

**Review by pavlo-mk — APPROVED** [review]

**Review by patryk-dabrowski — APPROVED** [review]

**Pull request merged**